tpp-private/style.css

```diff
@@ -76,21 +80,30 @@ header {
     cursor: pointer;
 }
 
+
 .addimg:hover {
     opacity: 0.5;
 }
 
-label {
+
+.adddoc label {
     cursor: pointer;
 }
 
+
+.addimg label {
+    cursor: pointer
+}
+
+
 .photoholder {
     display: flex;
     grid-column: 2/6;
     justify-content: space-around;
     flex-wrap: wrap;
 }
 
+
```

Comment:
camelCassing here again documentsTitle would be right way to do it

Reply:
done